x-pack/plugins/security_solution/public/detections/components/rules/step_schedule_rule/index.tsx

```diff
@@ -104,7 +107,7 @@ const StepScheduleRuleComponent: FC<StepScheduleRuleProps> = ({
               minimumValue: 1,
```

Comment:
`minimumValue: 1` is missing for the interval so it's possible to set 0 on the Rule Creation page:

<img width="461" alt="Screenshot 2022-09-19 at 13 59 30" src="https://user-images.githubusercontent.com/7359339/191012402-bca8748d-eede-4d2b-84f2-27fed4644b13.png">

But it won't allow you to create such a rule:

<img width="461" alt="Screenshot 2022-09-19 at 14 01 20" src="https://user-images.githubusercontent.com/7359339/191012746-1529423d-5cca-4330-b2a5-94b02a82bd23.png">

Could you please fix it in a separate PR after merging this one?

Reply:
Fixing in new PR